src/Engine/ProtoCore/Lang/FunctionGroup.cs

```diff
@@ -177,6 +178,26 @@ public void CopyPublic(List<FunctionEndPoint> rhs)
             return ret;
         }
 
+        internal static Dictionary<CLRFunctionEndPoint, int> GetConversionDistances(IEnumerable<CLRFunctionEndPoint> functionEndPoints,
```

Comment:
Are these `functionEndPoints` (plural) the overloaded methods? Can we simplify the replication logic by ignoring method overloads for the current phase of work?

Reply:
Yes, `functionEndPoints` (here) represent overaloaded methods. However, it will only contain a single functionEndpoint because overloaded methods are not analyzed yet.
Since I already took the time to convert these methods to the new VM...I would like to keep them...even though they do not do much for now